india_compliance/gst_india/print_format/gstr_3b/gstr_3b.html

```
@@ -0,0 +1,12 @@
+<style>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)